File: Baekjoon/solved_ac/class1/2920.cpp
// 2920: 음계

#include <iostream>
#include <string>

int main() {
  int chords[8];
  for (int i=0; i<8; i++)
    std::cin >> chords[i];

  std::string result = chords[0] < chords[1] ? "ascending":"descending";

  for (int i=1; i<8; i++) {
    result = (result == "ascending" && chords[i-1] > chords[i]) or (result == "descending" && chords[i-1] < chords[i]) ? "mixed":result;
  }

  std::cout << result;

  return 0;
}

File: Baekjoon/solved_ac/class1/2920.py
# 2920: 음계

chords = list(map(int, input().split()))

result = "ascending" if chords[0] < chords[1] else "descending"

for i in range(1, 8):
  result = "mixed" if (result == "ascending" and chords[i-1] > chords[i]) or (result == "descending" and chords[i-1] < chords[i]) else result

print(result)